Tests/SQLKitTests/SQLKitTests.swift

```diff
@@ -917,5 +929,19 @@ CREATE TABLE `planets`(`id` BIGINT, `name` TEXT, `diameter` INTEGER, `galaxy_nam
             select.column("id").from("t1")
         }.run().wait()
         XCTAssertEqual(db.results[18], "SELECT `id` FROM `t1`")
+
+        // Test that LIMIT, OFFSET, and ORDERBY are applied correctly
+        db._dialect.unionFeatures.remove(.explicitDistinct)
+        try db.select()
+            .column("id").from("t1")
+            .union({
+                $0.column("id").from("t2")
+            })
+            .limit(3)
+            .offset(5)
+            .orderBy("id")
```

Comment:
While I'm being pedantic, can you add a test that makes sure multiple `ORDER BY` columns are serialized correctly as well?

Reply:
Sure, I've added one.